server/branding.py

```
@@ -13,7 +13,10 @@
 from __future__ import annotations
 
 import re
-import xml.etree.ElementTree as ET
+# ‏defusedxml אינה נדרשת כאן, וההסבר המלא ב-`_svg_refusal`: השומר על
+# תוכן פעיל רץ **לפני** הפענוח ו-`_ACTIVE` כולל `<!ENTITY`. נמדד:
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)